app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -96,10 +96,9 @@ def center_toolbar_filename_explorer
         end
       elsif @layout == "miq_policy_rsop"
         return session[:rsop_tree] ? "miq_policy_rsop_center_tb" : "blank_view_tb"
-      elsif @layout == "provider_foreman"
-        if x_active_tree == :configuration_manager_providers_tree || :cs_filter_tree
-          return center_toolbar_filename_configuration_manager_providers
-        end
+      elsif @layout == "provider_foreman" && (x_active_tree == :configuration_manager_providers_tree ||
```

Comment:
I think this would be easier to read like this:
```ruby
elsif @layout == "provider_foreman" && [:configuration_manager_providers_tree, :cs_filter_tree, :configuration_scripts_tree].include?(x_active_tree)
```